manim/mobject/svg/text_mobject.py

```diff
@@ -575,3 +578,331 @@ def change_alignment_for_a_line(self, alignment, line_no):
                     ]
                 )
             )
+
+
+class PangoText(SVGMobject):
+    """Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
+
+    PangoText objects behave like a :class:`.VGroup`-like iterable of all characters
+    in the given text. In particular, slicing is possible.
+
+    Parameters
+    ----------
+    text : :class:`str`
+        The text that need to created as mobject.
+
+    Returns
+    -------
+    :class:`PangoText`
+        The mobject like :class:`.VGroup`.
+
+    Examples
+    --------
+    .. manim:: PangoRender
+        :quality: low
+
+        class PangoRender(Scene):
+            def construct(self):
+                morning = PangoText("வணக்கம்", font="sans-serif")
+                self.play(Write(morning))
+                self.wait(2)
+
+    .. WARNING::
+
+        Using a :class:`.Transform` on text with leading whitespace can look
+        `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
+        :meth:`remove_invisible_chars` to resolve this issue.
+
+    """
+
+    CONFIG = {
+        # Mobject
+        "color": WHITE,
+        "height": None,
+        "width": None,
+        "fill_opacity": 1,
+        "stroke_width": 0,
+        "should_center": True,
+        "unpack_groups": True,
+        # Text
+        "font": "",
+        "gradient": None,
+        "line_spacing": -1,
+        "size": 1,
+        "slant": NORMAL,
+        "weight": NORMAL,
+        "t2c": {},
+        "t2f": {},
+        "t2g": {},
+        "t2s": {},
+        "t2w": {},
+        "tab_width": 4,
+    }
+
+    def __init__(self, text: str, **config):
+        self.full2short(config)
+        digest_config(self, config)
+        self.original_text = text
+        text_without_tabs = text
+        if text.find("\t") != -1:
```

Comment:
Question: why is it necessary to replace the tabs? What if the user wants tabs?

Reply:
Pretty much this allow users to set custom length of tabs.  Also, tabs are added, by a default width of 4, you just try and see.